src/main/java/bio/terra/service/dataset/DatasetBucketDao.java

```diff
@@ -131,4 +143,16 @@ private void datasetBucketLinkUpdate(String sql, UUID datasetId, UUID bucketReso
         }
 
     }
+
+    private static class UuidMapper implements RowMapper<UUID> {
```

Comment:
I don't have it handy, but how do we handle the case today where we read UUIDs from the DB?